ProjFS.Mac/Scripts/Build.sh

```diff
@@ -83,7 +82,7 @@ while read line; do
 	 [[ $line != *".xctest"* ]] && 
 	 [[ $line != *".cpp"* ]] && 
 	 [[ $line != *".hpp"* ]]; then
-       printf "\nError: Not at 100% Code Coverage: $line"
+       echo "Error: not at 100% Code Coverage $line"
```

Comment:
The bash on the build machine as not able to execute the printf, so it's been updated to echo.  Locally of course it all ran fine.